catalyst_voices/packages/libs/catalyst_key_derivation/lib/src/bip32_ed25519/bip32_ed25519_public_key.dart

```diff
@@ -31,6 +32,10 @@ class Bip32Ed25519XPublicKey extends Equatable {
   /// Returns the bytes of the public key.
   List<int> get bytes => _bytes.inner;
 
+  /// Extracts the public key bytes from the extended public key.
+  Ed25519PublicKey extractPublicKey() =>
```

Comment:
I renamed the method to `toPublicKey`, didn't decide on `toEdPublicKey` because both Bip32Ed25519XPublicKey and Ed25519PublicKey are based on `Ed` curve, so it stays the same. 

The method should be understood as conversion from extended public key to just a public key.